<launch>
    
    <!-- <param name="central_bag_path" type="string" value="/home/ding/backup/dataset/haibo/0119s95.bag" /> -->
    <!-- <param name="query_bag_path" type="string" value="/home/ding/backup/dataset/haibo/0118_2s95.bag" /> -->
    <!-- <param name="query_bag_path" type="string" value="/home/ding/backup/dataset/haibo/0119filter.bag" /> -->

    <!-- <param name="central_bag_path" type="string" value="/home/ding/backup/dataset/haibo/data/0313_1/0313b95.bag" /> 
    <param name="query_bag_path" type="string" value="/home/ding/backup/dataset/haibo/data/0313/bb95_0313.bag" />
    <param name="central_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0313_1/map.pcd" />
    <param name="query_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0313/update/updated_map.pcd" />
    <param name="query_init_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0313/map.pcd" /> -->


    <param name="central_bag_path" type="string" value="/home/ding/backup/dataset/haibo/data/0414/0414good.bag" /> 
    <param name="query_bag_path" type="string" value="/home/ding/backup/dataset/haibo/data/0405/0405bad.bag" />
    <param name="central_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0414/map.pcd" />
    <param name="query_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0405/update/updated_map.pcd" />
    <param name="query_init_pcd_path" type="string" value="/home/ding/backup/dataset/haibo/data/0405/map.pcd" />

    <param name="imu_topic" type="string" value="/imu_data"/>
    <param name="lidar_topic" type="string" value="/lidar_aeb/raw_points"/>


    <node pkg="json_test" type="lt_visualize" name="lt_visualize" output="screen"/>

    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find json_test)/rviz/test3.rviz" />
    </group>

</launch>